packages/nimble-components/src/table-column/text/index.ts

```diff
@@ -22,6 +25,12 @@ declare global {
  * The table column for displaying string fields as text.
  */
 export class TableColumnText extends TableColumnTextBase {
```

Comment:
wait, just realized.. why are we not passing TableColumnTextColumnConfig to the parent class to fulfill the generic for the TableColumn base class? I'm guessing the typing of columnInternals isn't right.

May have been not a good idea to let TableColumn default to having ColumnConfig Type `unknown`.

Reply:
We aren't passing the generic type because of our mixins. TypeScript doesn't support mixins + generics (see https://github.com/microsoft/TypeScript/issues/26154).

I still playing around with the types, but I haven't found a way to get the generic passed from a concrete implementation (say TableColumnText) all the way to the TableColumn base class.

Reply:
Doesn't have to block the PR as discussed at stand-up 👍

Reply:
Created #1964